### PR TITLE
docs: add Ukranian to the project README

### LINKED ISSUE #5140: Ukrainian translation on Crowdin 100%
#### Is your feature request related to a problem? Please describe.
The Ukrainian translation on Crowdin has been completed 💯. Can you add it to the website? 🙏

### PR DESCRIPTION
**Pull request by matchai**
#### Description

Adds Ukranian to the project website and README

#### Motivation and Context

Closes #5140 

#### How Has This Been Tested?

- [ ] I have tested using **MacOS**
- [ ] I have tested using **Linux**
- [ ] I have tested using **Windows**

#### Checklist:

- [ ] I have updated the documentation accordingly.
- [ ] I have updated the tests accordingly.

### CODE REVIEW

File: docs/.vuepress/config.ts [hunk 1]
```diff
@@ -276,6 +281,24 @@ module.exports = defineConfig({
                     faq: "Часто Задаваемые Вопросы",
                 }),
             },
+            "/uk-UA/": {
+                // text for the language dropdown
+                selectText: "Переклади",
+                // label for this locale in the language dropdown
+                label: "Українська",
+                // Custom text for edit link. Defaults to "Edit this page"
+                editLinkText: "Edit this page on GitHub",
```
Comment (matchai) [inline docs/.vuepress/config.ts:290]: @Andygol would you be willing to suggest the Ukranian translation for this?

File: docs/.vuepress/config.ts [hunk 2]
```diff
@@ -276,6 +281,24 @@ module.exports = defineConfig({
...
+                // Custom navbar values
+                nav: [{ text: "Налаштування", link: "/uk-UA/config/" }],
+                // Custom sidebar values
+                sidebar: sidebar("uk-UA", {
+                    guide: "Guide",
```
Comment (matchai) [inline docs/.vuepress/config.ts:295]: @Andygol as well as this one.

I've extracted the other strings from elsewhere in the translations.

File: docs/.vuepress/config.ts [hunk 3]
```diff
@@ -276,6 +281,24 @@ module.exports = defineConfig({
                     faq: "Часто Задаваемые Вопросы",
                 }),
             },
+            "/uk-UA/": {
+                // text for the language dropdown
+                selectText: "Переклади",
```
Comment (Andygol) [inline docs/.vuepress/config.ts:286]:
```suggestion
                selectText: "Мови",
```

File: docs/.vuepress/config.ts (hunk 1 above)
Comment (Andygol) [inline docs/.vuepress/config.ts:290]:
```suggestion
                editLinkText: "Редагувати цю сторінку на GitHub",
```

File: docs/.vuepress/config.ts (hunk 2 above)
Comment (Andygol) [inline docs/.vuepress/config.ts:295]:
```suggestion
                    guide: "Керівництво",
```

File: docs/.vuepress/config.ts [hunk 4]
```diff
@@ -276,6 +281,24 @@ module.exports = defineConfig({
...
+                editLinkText: "Edit this page on GitHub",
+                // Custom navbar values
+                nav: [{ text: "Налаштування", link: "/uk-UA/config/" }],
+                // Custom sidebar values
+                sidebar: sidebar("uk-UA", {
+                    guide: "Guide",
+                    installing: "Розширене встановлення",
+                    config: "Налаштування",
+                    "advanced-config": "Розширені налаштування",
+                    faq: "Часті питання",
```
Comment (Andygol) [inline docs/.vuepress/config.ts:299]:
```suggestion
                    faq: "Часті питання",
                    presets: "Шаблони",
```

**Review by Andygol — COMMENTED** [review]
@matchai Here is my suggestion on adding Ukrainian language to the website.

**Pull request merged**